### PR TITLE
docs: add example for piping anonymous fn

### PR DESCRIPTION
**Pull request by rodesousa**
Signed-off-by: rodesousa <dessroberto@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -655,6 +655,14 @@ square = & &1 * &1
 
 The above style is much better than previous . You can also use `fn` to define anonymous functions.
 
+Or use `Kernel.then/2` function
```
Comment (blackode) [inline README.md]:
```suggestion
Or use `Kernel.then/2` function since 1.12.0
```

**Review by blackode — CHANGES_REQUESTED** [review]
Mention the elixir version as `then` macro is available from `1.12.0`.

File: README.md (hunk 1 above)
Comment (rodesousa) [reply README.md]: done !

**Comment by blackode** [comment]
@all-contributors add @rodesousa for code

**Pull request merged**